src/dispatch-payload.ts

```diff
@@ -40,16 +40,9 @@ export const DispatchPayload = async (
     throw new Error("Could not find the function callback_id in the payload");
   }
 
-  // Let caller resolve the function directory
-  const potentialFunctionFiles = getFunctionFiles(functionCallbackId);
-  const functionModule = await LoadFunctionModule(potentialFunctionFiles);
-  if (!functionModule) {
-    throw new Error(
-      `Could not load function module for function: "${functionCallbackId}" in any of the following locations: \n${
-        potentialFunctionFiles.join("\n")
-      }\nMake sure your function's "source_file" is relative to your project root.`,
```

Comment:
The passed-in `getFunctionFile` callback, which is provided by either `local-run.ts` (local-run context) or `mod.ts` (remote-run context), always returns either a string or a full function module. The result of that is then fed through `LoadFunctionModule`, which either `import`s it if the result was a string, or returns the function module directly.

As a result, I feel like we don't need this extra check for falsy returns from `LoadFunctionModule`. If there was an  error during `import`, then that will throw with a "Module not found" error, which hopefully is obvious enough to the user. I feel like less is more in this situation.